src/Modules/OrchardCore.Commerce.Payment.Stripe/Abstractions/IStripePaymentService.cs

```diff
@@ -61,4 +61,11 @@ public interface IStripePaymentService
         Controller controller,
         ContentItem order,
         PaymentIntent paymentIntent);
+
+    /// <summary>
+    /// Get the confirmation parameters for Stripe.
+    /// </summary>
+    /// <param name="middlewareUrl">The url for the middleware of Stripe.</param>
+    /// <returns><see cref="PaymentIntentConfirmOptions"/>.</returns>
+    Task<PaymentIntentConfirmOptions> GetStripeConfirmParametersAsync(string middlewareUrl);
```

Comment:
Shouldn't the parameter type be `Uri` instead?

Reply:
not necessary in this case, and not increase complicated

Reply:
After looking the code over again, this has to be absolute URL. Normally in ASP.Net Core you pass around relative virtual URLs, like `~/checkout/middleware/Stripe`. Which wouldn't work in this case because the full URL is sent out to a remote server.
Please rename the parameter to `middlewareAbsoluteUrl`

Reply:
renamed to middlewareAbsoluteUrl